applications/ContactStructuralMechanicsApplication/python_scripts/contact_structural_mechanics_implicit_dynamic_solver.py

```diff
@@ -92,18 +94,18 @@ def AddDofs(self):
         super().AddDofs()
         
         if (self.contact_settings["mortar_type"].GetString() == "ALMContactFrictionless"):
```

Comment:
Should this be a Dog?

Reply:
¿?

Reply:
Dog¿?

Reply:
i am asking if you need this variable to be a dof

Reply:
NORMAL_CONTACT_STRESS?, of course, is the LM

Reply:
ok, then merge